plugins/sentry/src/api/production-api.ts

```diff
@@ -47,4 +50,16 @@ export class ProductionSentryApi implements SentryApi {
 
     return (await response.json()) as SentryIssue[];
   }
+
+  async authOptions() {
```

Comment:
Should mark this as private, so that it disappears from the api-report.

Reply:
@freben Fredrik thanks a lot - and sorry, completely missed this is inside the exported class. should be better now (just it would be better to squash if merging).